wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -43,11 +43,10 @@ fun getIcon(icon: String?, domain: String, context: Context): IIcon? {
     return getIcon(simpleEntity as Entity<Map<String, Any>>, domain, context)
 }
 
-fun getIcon(entity: Entity<Map<String, Any>>?, domain: String, context: Context): IIcon? {
+fun getIcon(entity: Entity<Map<String, Any>>?, domain: String, context: Context?): IIcon? {

```

Comment:
Same question as for the other function, why are you changing `Context` to `Context?` here? `LocalContext.current`,  which is what is passed in, should always be non-null.

Reply:
I seem to have misunderstood the use of `?` in Kotlin, a beginner mistake, I assume. Fixing.